services/QuillLMS/engines/comprehension/spec/dummy/app/models/activity_classification.rb

```diff
@@ -1,4 +1,4 @@
-class ActivityClassification < ApplicationRecord
+class ActivityClassification < ActiveRecord::Base
```

Comment:
I thought that `ApplicationRecord` was the new Rails 5 approach.  Do we want to go back to the old way here?

Reply:
Good catch - this PR was created before Rails 5.1 was merged, and the rebase did not clobber the old because I used `cp -R` to create `spec/` from `test/`